src/components/Suggestions.ts

```diff
@@ -105,3 +105,41 @@ export function getSuggestions(model: monacoTypes.editor.ITextModel, position: m
     suggestions: [],
   };
 }
+
+export function getSignatureHelp(
+  model: monacoTypes.editor.ITextModel,
+  position: monacoTypes.Position
+): monacoTypes.languages.ProviderResult<monacoTypes.languages.SignatureHelpResult> {
+  const word = model.getWordUntilPosition(position.delta(0, -1));
```

Comment:
hi! mind talking through what this is doing I'm not sure I follow why we need word or what word this represents and how this is different from textUntilPosition? Couldn't we just pass 0 for the startColumn or am I misunderstanding?

Reply:
true,  `word` may be misleading, this is actually the `wordPosition`. I just noticed that a word position actually has the property `word` (which is the word text) so I can highly simplify this :)